Subject: Use correct path for myspell dictionaries
 This patch fixes the myspell path to the dictionaries
 installed on the system for Debian.
Author: Sebastian Reichel <sre@debian.org>
Forwarded: not-needed

diff --git a/src/options.cpp b/src/options.cpp
index a0fd722..5190847 100644
--- a/src/options.cpp
+++ b/src/options.cpp
@@ -121,7 +121,7 @@ void OptionsManager::LoadDefaults(bool onlyDefaults,bool doOverride) {
 #if defined(__WINDOWS__) || defined(__APPLE__)
 	SetText(_T("Dictionaries path"),_T("?data/dictionaries"));
 #else
-	SetText(_T("Dictionaries path"),wxString::Format(_T("%s/%s"), _T(INSTALL_PREFIX),_T("/share/myspell")));
+	SetText(_T("Dictionaries path"),wxString::Format(_T("%s/%s"), _T(INSTALL_PREFIX),_T("/share/myspell/dicts")));
 #endif
 	SetText(_T("Spell Checker"),_T("hunspell"));
 	SetModificationType(MOD_AUTOMATIC);
